exporter/internal/app/middleware.go

```diff
@@ -160,6 +160,21 @@ func ForwardCookie(name string) func(next http.Handler) http.Handler {
 	}
 }
 
+func ForwardSSOCookie(name string, as string) func(next http.Handler) http.Handler {
```

Comment:
I suggest we do `func ForwardCookieRenamed(name string, as string)` and `ForwardCookie` under the hood will call it with same name in 2 args. So we don't copy-paste without need.